<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radial Basis Function Network</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="gPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="errorPlot" native="true">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>491</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="paramsGB">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>202</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Parámetros</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="kLbl">
       <property name="text">
        <string>K-Nodos:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="kSB"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lrLbl">
       <property name="text">
        <string>Learning Rate:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="funcLbl">
       <property name="text">
        <string>Función:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="funcCB">
       <item>
        <property name="text">
         <string>y = sin(x)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="stepLbl">
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="stepSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="lrSB"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="initBtn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>490</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicializar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trainBtn">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>490</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Entrenar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
